docs/manual/troubleshooting.tex

```diff
@@ -953,25 +953,6 @@
 \end{Verbatim}
 
 
-\subsectionAndLabel{Related repositories}{building-related-repositories}
-
-Building the Checker Framework depends on other repositories, such as
-\href{https://github.com/typetools/annotation-tools/}{Annotation Tools} and
-\href{https://github.com/typetools/stubparser/}{Stub Parser}.
```

Comment:
This paragraph seems to still be true, except for the line about Annotation Tools.  The paragraph could mention the JDK.